src/Page/Food/Builder.elm

```diff
@@ -100,9 +104,21 @@ update : Session -> Msg -> Model -> ( Model, Session, Cmd Msg )
 update session msg model =
     case msg of
         AddIngredient { mass, code } ->
+            let
+                processCode =
+                    if model.isIngredientOrganic then
+                        session.foodDb.ingredients
+                            |> List.filter (.conventional >> .code >> (==) code)
+                            |> List.head
+                            |> Maybe.map (.labels >> .organic >> Maybe.map .code >> Maybe.withDefault code)
+                            |> Maybe.withDefault code
+
+                    else
+                        code
```

Comment:
This `if` will be replaced by a `case` as soon as I rework the format of the `ingredients.json` file to have all the different labels at the same level, instead of treating the "conventional/default"  differently.